playbilling/src/main/java/com/google/androidbrowserhelper/playbilling/digitalgoods/BillingResultMerger.java

```diff
@@ -35,26 +35,30 @@
  * {@link #setInAppResult} and {@link #setSubsResult}) should both take place on the UI thread, so
  * we don't need to worry about any concurrency.
  */
-public class BillingResultMerger {
-    private final SkuDetailsResponseListener mOnCombinedResult;
+public class BillingResultMerger<T> {
+    private final ResultListener<T> mOnCombinedResult;
 
     private @Nullable BillingResult mInAppResult;
-    private @Nullable List<SkuDetails> mInAppDetailsList;
+    private @Nullable List<T> mInAppDetailsList;
```

Comment:
Will member names such as this, and arguments elsewhere, also be changed? I'd expect mInAppDetailsList to always be a List<SkuDetails> for example.

Reply:
It is always going to be a list (the type is `List<T>`)...

Although I did change the names to get rid of "Details", so it's now `mInAppResultsList`.